drivers/spi/spi_ll_stm32.meta.jinja2

```diff
@@ -0,0 +1,36 @@
+{# Copyright (c) 2018 Nordic Semiconductor ASA #}
+{# SPDX-License-Identifier: Apache-2.0         #}
+
```

Comment:
I assume we cannot have this file as part of `i2c_ll_stm32_device_template.h` and rename it to `i2c_ll_stm32_device_template.h.jinja2`?

Reply:
@carlescufi , if we're adding the .jinja2 extension, do we really need the "_template" suffix?
I think it'll be more user friendly if we can contain a bit the complete file name..

Reply:
Because how jinja2 technically works cannot a template that inherit another template (by using the extends keyword) be rendered directly, but must be imported by another template. But I kind of think that separating out the metadata is cleaner anyway.

This is also related to how I decided to solve this particular use-case, it can be solved in many ways. I could for instance have made a similar solution to how the COG version do it by using the built-in replace filter, but that would resulted in generator within the generator. I could also have exposed python functions through a data object, and that is maybe something we should look into in the long term, but for now I wanted to show a less complex jinja2 only solution.

The naming scheme is maybe a bad adaptation of your request to avoid the .jinja2 extension. I needed a different name on the file than the outputted file so that's why I introduced the _template postfix. The outputted file is called i2c_ll_stm32_device.h